### PR TITLE
{bio}[dummy/] RDP-Classifier v2.12

### PR DESCRIPTION
**Pull request by FokkeDijkstra**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/RDP-Classifier/RDP-Classifier-2.12-Java-1.8.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['rdp_classifier_%(version)s.zip']
+checksums = ['977896248189a1ce2146dd3a61d203c3c6bc9aa3982c60332d463832922f7d0a']
+
+java = 'Java'
+javaver = '1.8'
+versionsuffix = '-%s-%s' % (java, javaver)
+
+dependencies = [(java, javaver)]
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/r/RDP-Classifier/RDP-Classifier-2.12-Java-1.8.eb]: @FokkeDijkstra ping?

[the remark this answers is not in the document]
Comment (FokkeDijkstra) [reply easybuild/easyconfigs/r/RDP-Classifier/RDP-Classifier-2.12-Java-1.8.eb]: Apologies, forgot about this one. I now made the changes you requested.

[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/r/RDP-Classifier/RDP-Classifier-2.12-Java-1.8.eb]: No need to apologize, I'm well aware how easy it is to overlook PRs... ;)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/6b902ca05e3258daae0b5292e9446d5b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2005.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4d635e7d007856b4837d6ff28472fa1f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2524.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3996989fc60dfa0039148826113a6ead for a full test report.

**Comment by boegel** [comment]
Going in, thanks @FokkeDijkstra!

**Pull request merged**